src/Concretize.hs

```diff
@@ -1262,6 +1268,15 @@ manageMemory typeEnv globalEnv root =
                   Right (XObj (Lst (okF : okArgs)) i t)
         [] -> pure (Right xobj)
     visitList _ = error "Must visit list."
+    searchBreak :: Set.Set Deleter -> XObj -> XObj
```

Comment:
I'm confused by this function's name, is there perhaps an alternative that fits it better? Or maybe I just need to be enlightened.

Reply:
`searchForInnerBreak` might be better?

Reply:
Or `fixupInnerBreak`?

Reply:
searchForInnerBreak is more clear in my mind

Reply:
Renamed!